x-pack/plugins/ml/public/application/jobs/new_job/pages/components/charts/event_rate_chart/overlay_range.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
```

Comment:
Try import from `@elastic/eui`

Reply:
Changed in ac22c8ce86162fd9d4db4687850de4ae33873f4b